### DISCUSSION TITLE
Error when launching Visual Studio Code: "Language Support for Java (Syntax Server) client: couldn't create connection to server."

### DISCUSSION THREAD

**Discussion by RomainThepaut** [microsoft/vscode-discussions#1141] [Extension Development QnA]
When I try to launch Visual Studio Code, I encounter an error indicating that the Java language support server could not be started. The full error message is: "[Error - 9:06:16 AM] Language Support for Java (Syntax Server) client: couldn't create connection to server. Launching server using command c:\Users\rmthe.vscode\extensions\redhat.java-1.29.0-win32-x64\jre\17.0.10-win32-x86_64\bin\java failed. Error: spawn c:\Users\rmthe.vscode\extensions\redhat.java-1.29.0-win32-x64\jre\17.0.10-win32-x86_64\bin\java ENOENT".

This error seems to indicate an issue with establishing a connection to the Java server. It appears that Visual Studio Code is attempting to launch Java from a path that does not exist, resulting in an "ENOENT" (No such file or directory) error.

I have recently updated my Java version on my system, and this may have affected the configuration of Visual Studio Code. I am seeking a solution to resolve this issue and enable me to use Visual Studio Code with the new Java version. If anyone has encountered this problem before or has any suggestions for resolving it, I would appreciate any help.

<img width="959" alt="Capture d'écran 2024-04-16 062338" src="https://github.com/microsoft/vscode-discussions/assets/96549983/0cc1f4c3-132f-4abc-a0ad-b60434a9d410">

**Comment by abrilurena** [comment]
Hi, @RomainLEcrack! hope you're doing great. This is such a great question, it's the very first time I have seen this - I'm adding @isidorn as he might be able to help here :D

**Comment by isidorn** [comment]
Hi @RomainLEcrack - it looks like you are a user of the java extension. And this community is for vscode extension authors.

Thus I will go ahead and close this issue. And I suggest you open a question on Stack Overflow, or to open an issue in this repo https://github.com/redhat-developer/vscode-java
